frameworks/hdfs/tests/test_shakedown.py

```diff
@@ -318,62 +308,3 @@ def replace_name_node(index):
     tasks.check_tasks_updated(PACKAGE_NAME, name_node_name, name_id)
     tasks.check_tasks_not_updated(PACKAGE_NAME, 'journal', journal_ids)
     tasks.check_tasks_not_updated(PACKAGE_NAME, 'data', data_ids)
-
```

Comment:
moved to the config.py in that test dir, because it's used by the overlay and non-overlay tests